docker-for-windows/release-notes.md

```diff
@@ -82,6 +82,27 @@ Docker Desktop Community 2.1.0.0 contains the following experimental features:
 * When two services have a common exposed port, Docker Desktop exposes the available ports for the second service.
 * Fixed the UI lock when changing the Kubernetes state.
 
+### Known issues
+
+Attempts to upgrade Docker Desktop to version 2.1.0.0 can fail with the error `value cannot be null`. To work around this issue:
+
+1. Open the `settings.json` file located at `%APPDATA%\Docker`.
```

Comment:
I would be more precise here with
%APPDATA%\Docker\settings.json (e.g. C:\Users\USERNAME\AppData\Roaming\Docker\settings.json)

Reply:
Thanks. I've updated the location.